apollo-coroutines-support/src/main/kotlin/com/apollographql/apollo/coroutines/CoroutinesExtensions.kt

```diff
@@ -14,7 +14,11 @@ import kotlin.coroutines.resumeWithException
 private class ChannelCallback<T>(val channel: Channel<Response<T>>) : ApolloCall.Callback<T>() {
 
     override fun onResponse(response: Response<T>) {
-        channel.offer(response)
+        try {
+            channel.offer(response)
+        } catch (ex: Exception) {
+            // Swallow exception if the channel is closed.
```

Comment:
Can this catch block be more specific and not catch all Exceptions ?

Reply:
It's possible to just catch `ClosedSendChannelException` (in case the channel is closed without cause) and `ApolloException` (in case the channel is closed by `onFailure(e: ApolloException)`)